Add delete site from Cloudflare section (#2374)
cloudflare/cloudflare-docs: products/pages/src/content/get-started/index.md

</Aside>

### Deleting a custom domain

> The Cloudflare dashboard has a list of services that your pages project can leverage on, such as Analytics, speed, and, SSL/TLS. When trying to remove your project from Cloudfalre you will not be able to just delete the project from the pages dashboard, if your DNS is handled by Cloudflare. You will have to delete custom domains before deleting the project from Pages dashboard. 

Reviewer: ```suggestion
If your DNS is handled by Cloudflare, you must delete any custom domain associated with your Pages project before removing your project from the **Pages** dashboard.  
```

> The dashboard for your site is reachable from the drop down menu at top right of the page.
> 

Reviewer: ```suggestion
Log in to the Cloudflare dashboard and use the **Account Home** or drop-down menu to select your website.
```

![drop down menu](./images/drop-down-menu.png)

> 
> On the dashboard overview page you have to scroll down to the advanced action option
> and click on **remove site from Cloudflare** this will bring up a modal and if you confirm, your site will be deleted from Cloudflare.

Reviewer: ```suggestion
After you select your website, you will be directed to **Overview** on the dashboard. On **Overview**:
1. Scroll down to **Advanced Actions**.
2. Select **Remove Site from Cloudflare**.
3. To delete your website from Cloudflare, select **Confirm**.
```

![advanced-option](./images/advanced-option.png)

> Now if you return to the pages dashboard and go to **settings** > **delete project**. You can now remove your project from Cloudflare pages. 

Reviewer: ```suggestion
Go back to the **Account Home** or use the drop-down menu at the top of the dashboard and select **Pages**. You can now remove your project by selecting your **Pages project** > **Settings** > **Delete project**.
```
